talentlink-connector-product/product.json

```diff
@@ -22,14 +22,14 @@
         ]
       }
     },
-    {
-      "id": "maven-dependency",
+        {
+      "id": "maven-import",
       "data": {
-        "dependencies": [
+        "projects": [
           {
-            "groupId": "MY-GROUP-ID",
-            "artifactId": "MY-PRODUCT-NAME",
-            "version": "${version}",
+            "groupId": "com.axonivy.connector.talentlink.demo",
```

Comment:
groupId seems to be wrong: should be "com.axonivy.connector.talentlink"

- [ ] fix groupId